index.js

```diff
@@ -7,35 +7,32 @@ const TIMEOUT_DEFAULT = 300;
 
 main();
 
-const getRegistryURL = (registry, package) => {
+function getRegistryURL(registry, package) {
   const sanitizedPackageName = package.replace(/\//, "%2f");
   const registryURL = new URL(sanitizedPackageName, registry);
-  
+
   return registryURL.toString();
 }
 
 async function main() {
   try {
     const package = core.getInput("package");
     const version = core.getInput("version").replace(/^v/, "");
-    const timeout = core.getInput("timeout") | TIMEOUT_DEFAULT;
-    const registry = core.getInput("registry") | "https://registry.npmjs.org"
+    const timeout = core.getInput("timeout") || TIMEOUT_DEFAULT;
```

Comment:
@gr2m this fixes #43 
It was doing a logical OR instead of a fallback, so it was evaluating to `0`

Reply:
sorry I missed that!